frontend/src/components/Form/PaginatedSelectMenu.tsx

```diff
@@ -41,13 +41,19 @@ export interface PaginatedSelectMenuProps<
   required?: boolean;
   /** Text that is displayed instead of the input if it has not been selected yet.*/
   placeholder?: string;
+  /** Maximum time interval between two inputs (in milliseconds) before they are considered separate inputs. */
   debounceTimeout?: number;
+  /** Options controlling how data is loaded by react-select. */
```

Comment:
Can you add a link to more information about this?

Reply:
I could not find a fitting link that would explain this type.
However I have clarified the documentation a bit.